packages/components/src/components/Accordion/Accordion.tsx

```diff
@@ -0,0 +1,84 @@
+import type { ComponentProps, FC, PropsWithChildren, ReactNode } from "react";
+import React, { useId, useState } from "react";
+import clsx from "clsx";
+import styles from "./Accordion.module.scss";
+import type { PropsContext } from "@/lib/propsContext";
+import { dynamic, PropsContextProvider } from "@/lib/propsContext";
+import { Button } from "@/components/Button";
+import { IconChevronDown } from "@/components/Icon/components/icons";
+import { TunnelExit, TunnelProvider } from "@mittwald/react-tunnel";
+
+export interface AccordionProps
+  extends PropsWithChildren<ComponentProps<"div">> {
+  defaultExpanded?: boolean;
+}
+
+export const Accordion: FC<AccordionProps> = (props) => {
+  const { children, className, defaultExpanded = false, ...rest } = props;
+  const [expanded, setExpanded] = useState(defaultExpanded);
+
+  const rootClassName = clsx(
+    styles.accordion,
+    expanded && styles.expanded,
+    className,
+  );
+
+  const headerId = useId();
+  const contentId = useId();
+
+  const headerButton = (children: ReactNode) => (
+    <Button
+      unstyled
+      aria-expanded={expanded}
+      className={styles.headerButton}
+      onPress={() => setExpanded(!expanded)}
```

Comment:
```suggestion
      onPress={(expanded) => setExpanded(!expanded)}
```
Siehe https://react.dev/reference/react/useState#updating-state-based-on-the-previous-state

Reply:
aber das müsste dann eher so sein oder?
`  onPress={() => setExpanded((expanded) => !expanded)}`